File: db4on/core/src/core/com/db4o/QEEqual.cs
namespace com.db4o
{
	/// <exclude></exclude>
	public class QEEqual : com.db4o.QEAbstract
	{
		public override void IndexBitMap(bool[] bits)
		{
			bits[com.db4o.QE.EQUAL] = true;
		}
	}
}

File: db4oj/core/src/com/db4o/inside/QEEqual.java
/* Copyright (C) 2004   db4objects Inc.   http://www.db4o.com */

package com.db4o.inside;



/**
 * @exclude
 */
public class QEEqual extends QEAbstract
{
    public void indexBitMap(boolean[] bits){
        bits[QE.EQUAL] = true;
    }
}
